src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -49,6 +53,14 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
         return list(getattr(client, func)(container_name))
     return completer
 
+def entity_completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-argument
+    # This is a workaround for the fact that argcomplete always inserts a space after completion
+    # In this case, we want the cursor to remain positioned just after the text. We would ideally
+    # like it to append the = sign, but argcomplete irritatingly escapes it.
+    if prefix == 'RowKey' or prefix == 'PartitionKey':
```

Comment:
>if prefix == 'RowKey' or prefix == 'PartitionKey': [](start = 4, length = 50)

suggestion: prefix in ['RowKey', 'PartitionKey'] instead of the or condition.